### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by yakshaG**
## Changes
- Added content for home page
- Responsive home page

## Need help with
- Applying `Roboto Slab` and `Kalam` font properly, instead of inline CSS
- `<strong>` (bold) not working on chrome in Mac
- Converting `PersonCard.jsx` and `FeatureCard.jsx` to typescript
- Layout difference in local and deployed version — see screenshot below. Something to do with grid cols and margin bottom.

**Local:**
image

**Deployed (expected behaviour):**
image

### CODE REVIEW

File: src/templates/professional/components/Education.tsx [hunk 1]
```diff
@@ -11,11 +13,7 @@ export const Education = ({ education }: { education: EducationIntrf[] }) => {
             </p>
             <div className="flex justify-between items-center">
               <p className="font-normal text-[#1AB0B3]">{item.institution}</p>
-              <div className="flex gap-3">
-                <p className="text-xs">
-                  {item.startDate} - {item.endDate}
-                </p>
-              </div>
+              <p className="gap-3 text-xs">{`${item.startDate} - ${item.endDate}`}</p>
```
Comment (siva-kannan3) [inline src/templates/professional/components/Education.tsx:16]: This file has been updated. Can you pull from v2 branch and update this PR?

Comment (yakshaG) [reply src/templates/professional/components/Education.tsx]: Thanks. This is done.

**Review by siva-kannan3 — APPROVED** [review]

File: src/modules/home/HomeLayout.tsx [hunk 2]
```diff
@@ -1,15 +1,178 @@
...
+import PersonCard from './components/PersonCard';
+import { Button } from '@mui/material';
+import 'animate.css';
+import { useEffect } from 'react';
 
 const HomeLayout = () => {
+  useEffect(() => {
+    const element = document.querySelector('#resume-3d')!;
+
+    element.addEventListener('mouseenter', function () {
```
[the remark this answers is not in the document]
Comment (siva-kannan3) [reply src/modules/home/HomeLayout.tsx]: removed listener on unmount

File: src/templates/professional/components/Education.tsx [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import { EducationIntrf } from 'src/stores/index.interface';
 
 export const Education = ({ education }: { education: EducationIntrf[] }) => {
+  console.log(education[0].startDate, '--------');
```
[the remark this answers is not in the document]
Comment (siva-kannan3) [reply src/templates/professional/components/Education.tsx]: removed

**Review by siva-kannan3 — APPROVED** [review]

**Pull request merged**